bugalink-frontend/hooks/useReviews.ts

```diff
@@ -5,7 +5,7 @@ export default function useReviews(driverId) {
   // TODO: once we have local sessions, we can extract the user id from the session
```

Comment:
We have local sessions now (`useSession()` hook, check other hooks that use it), in case you need to extract the logged user's session. In case you don't, it's fine :)